AdminWebsite/AdminWebsite/ClientApp/src/app/shared/select/select.component.html

```diff
@@ -1,7 +1,7 @@
 <div>
-  <label class="govuk-label" for="select-menu">{{ title }}</label>
+  <label class="govuk-label" for="select-menu{{title}}">{{ title }}</label>
```

Comment:
I think you're missing a hyphen here - it should match the ng-select's id below

`select-menu-{{title}}`

Reply:
thanks fixed.